multistateview/src/main/java/com/hmos/compat/utils/AttrUtils.java

```diff
@@ -228,8 +231,50 @@ public static Element getElementFromAttr(AttrSet attrs, String name) {
                 value = attrs.getAttr(name).get().getElement();
             }
         } catch (Exception e) {
-            HiLog.error(HI_LOG_LABEL, TAG,e);
+            HiLog.error(HI_LOG_LABEL, TAG, e);
         }
         return value;
     }
+
+    /**
+     * Utils.
+     *
+     * @param attrs attrs.
+     * @return resIdEmpty.
+     */
+    public static Integer utils(AttrSet attrs) {
```

Comment:
utis1, utils2, utils3 is still duplication code. Have only one method and pass attr key as parameter.